boxoffice/views/order.py

```diff
@@ -197,6 +198,7 @@ def payment(order):
                 line_item.discount_coupon.update_used_count()
                 db.session.add(line_item.discount_coupon)
         db.session.commit()
+        boxofficeq.enqueue(stats.post_stats, order.item_collection_id, "Paid tickets")
```

Comment:
I would suggest an `if` condition here to check if the organization has Slack setup, before adding it to the queue.